chapters/3-package-information.md

```diff
@@ -405,11 +405,11 @@ Example:
     </Package>
 
     <Package rdf:about="...">
-        <downloadLocation rdf:resource="http://spdx.org/rdf/terms#noassertion"/>
+        <downloadLocation rdf:resource="spdx:noassertion"/>
```

Comment:
This is correct - these are the changes will improve the consistency in the documentation